common/dds-model.h

```diff
@@ -0,0 +1,66 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+#pragma once
+
+#include <librealsense2/rs.hpp>
+#include <imgui.h>
+#include <realsense_imgui.h>
+#include <set>
+#include <rsutils/type/ip-address.h>
+#include <rsutils/string/hexdump.h>
+#include <../third-party/rsutils/include/rsutils/type/eth-config.h>
+#include <rsutils/json.h>
+#include <rsutils/json-config.h>
+
+uint32_t const GET_ETH_CONFIG = 0xBB;
```

Comment:
Constsants should be under a namespace at minimum. Better yet, move to CPP as these are internal details that don't need to be included by users